requirements.txt

```diff
@@ -93,10 +91,8 @@ flask-sqlalchemy==3.0.3
     #   funding-service-design-utils
 flipper-client==1.3.1
     # via funding-service-design-utils
-funding-service-design-utils==2.0.41
+funding-service-design-utils==2.0.47
     # via -r requirements.in
-greenlet==3.0.1
```

Comment:
I guess we are removing some unwanted dependencies in here

Reply:
It happens automatically when we run precompile command